viewer/packages/360-images/src/Image360Icon.ts

```diff
@@ -9,18 +9,18 @@ export class Image360Icon extends THREE.Group {
   private readonly MAX_PIXEL_SIZE = 256;
   private readonly _hoverSprite: THREE.Sprite;
   private readonly _outerSprite: THREE.Sprite;
-  constructor(modelMatrix: THREE.Matrix4) {
```

Comment:
Sorry if I was unclear - I just meant that `model` in the name here might be perceived as the transform of the full CAD/Pointcloud model, and not the 360 image entity.. I was imagining something along the lines of `image360WorldTransform`. Do you think that makes it more clear?